client/src/App.test.js

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import App from './App';
+import * as rtl from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Navbar from './components/Navbar';
+import { useDarkMode } from './hooks/useDarkMode';
+import { useLocalStorage } from './hooks/useLocalStorage';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+test('NavBar renders without crashing', () => {
+	const wrapper = rtl.render(<Navbar />);
+	console.log(wrapper.debug());
+});
+test('App renders without crashing', () => {
+	const wrapper = rtl.render(<App />);
+	console.log(wrapper.debug());
+});
+test('DarkMode renders without crashing', () => {
```

Comment:
Dark Mode is a hook, not a component. It needs to be tested a bit differently since it doesn't actually render anything on its own.